src/actions.js

```diff
@@ -0,0 +1,82 @@
+export const toLogIn = () => {
+  return {
+    type: 'TO_LOG_IN'
+  }
+}
+
+export const toSignUp = () => {
+  return {
+    type: 'TO_SIGN_UP'
+  }
+}
+
+export const toProfiles = () => {
+  return {
+    type: 'TO_PROFILES'
+  }
+}
+
+export const setAUser = (UID) => {
+  return {
+    type: 'SET_USER',
+    UID
+  }
+}
+
+export const firstName = (e) => {
+  return {
+    type: 'ACCEPT_FIRST_NAME',
+    entry: e.target.value
+  }
+}
+export const lastName = (e) => {
+  return {
+    type: 'ACCEPT_LAST_NAME',
+    entry: e.target.value
+  }
+}
+export const email = (e) => {
+  return {
+    type: 'ACCEPT_EMAIL',
```

Comment:
You have `'ACCEPT_EMAIL'` typed out here and also later in `reducers.js`. What happens if you have a typo here and accidentally put in `'ACEPT_EMAIL'` ? You'd have a bug that the linter wouldn't find and that would be difficult to track down. I find that it's a better practice to define your constants at the top of this file and export them, e.g.:

```js
export let ACCEPT_LAST_NAME = 'ACCEPT_LAST_NAME'
...

export const email = (e) => {type: ACCEPT_LAST_NAME, entry}
```

then in your reducer, you would import `ACCEPT_LAST_NAME` from `actions`. That way, if anyone were to make a typo as above, it would be easily caught by the linter.